src/__experimental__/components/date/date.stories.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { action } from '@storybook/addon-actions';
+import { StateDecorator, Store, State } from '@sambego/storybook-state';
+import {
+  boolean,
+  number,
+  text
+} from '@storybook/addon-knobs';
+import DateInput from './date.component';
+import Textbox from '../textbox';
+import getTextboxStoryProps from '../textbox/textbox.stories';
+import notes from './documentation/notes.md';
```

Comment:
There seems to be a push to use an `index.js` in the documentation for Storybook.

Reply:
fixed